RecoEgamma/EgammaPhotonAlgos/src/InOutConversionSeedFinder.cc

```diff
@@ -205,6 +205,7 @@ void InOutConversionSeedFinder::fillClusterSeeds() const {
     }
     
     //PropagatorWithMaterial reversePropagator(oppositeToMomentum, 0.000511, &(*theMF_) );
+    assert(myPointer);
```

Comment:
We decided that, since this condition is so unlikely, a simple `assert` is good enough.